falcon/constants.py

```diff
@@ -141,6 +137,28 @@
     ]
 )
 
+# NOTE(vytas): We strip the preferred charsets from the default static file
+#   type mapping as it is hard to make any assumptions without knowing which
+#   files are going to be served. Moreover, the popular web servers (like
+#   Nginx) do not try to guess either.
+_DEFAULT_STATIC_MEDIA_TYPES = tuple(
+    (ext, media_type.split(';', 1)[0])
+    for ext, media_type in (
+        ('.bmp', MEDIA_BMP),
+        ('.gif', MEDIA_GIF),
+        ('.htm', MEDIA_HTML),
+        ('.html', MEDIA_HTML),
+        ('.jpeg', MEDIA_JPEG),
+        ('.jpg', MEDIA_JPEG),
+        ('.js', MEDIA_JS),
```

Comment:
do we want to add also mjs/cjs?
not that popular to have them served though

Reply:
Hm, not sure. Maybe we can start small with the most popular ones :grin:
`.cjs` is not even there: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types (`.mjs`, OTOH, is there).

Reply:
I think cjs is node only, so unknown by the browsers. mjs is instead support also by the browser.

It seems that a limited number of server use the js mime for that type already: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#aside_%E2%80%94_.mjs_versus_.js

I think we can go either way for this, so as you prefer

Reply:
We can add `.mjs`